### PR TITLE
Sidebar elements

### PR DESCRIPTION
**Pull request by jamesscottbrown**
**What this does**/**Why?**

This adds components for structuring the contents of sidebars, so that they are formatted more consistently.

**Does this introduce new dependencies?**

No.

**Is it tested?**

Only via examples in storybook.

**Is it documented?**

**Are light and dark themes considered?**

No.

**Have you included changeset file?**

Yes.

### CODE REVIEW

**Comment by jamesscottbrown** [comment]
@ChrisKnightLDN -hopefully this matches your Figma mockups fairly closely

@PaulioRandall - I'm curious how this compares to what you've been doing on sidebars.

**Comment by PaulioRandall** [comment]
@jamesscottbrown Yes, it's complimentary, not conflicting. [Draft PR](https://github.com/Greater-London-Authority/ldn-viz-tools/pull/106). Basically, implementing the various sidebar designs is not a concern of my PR.

**In a nutshell**: I've been working on positioning and behavior of the sidebar in such a way that the visual design of toggles, tabs, and content can be created and maintained separately (slotted in). Since it's a layout component I had to make a few default content components so I have something to show in storybook. I'm also copying over the _tabbedSidebar_ code, initially for demonstration purposes but I'm hoping we'll modify it as our Figma designs change and stabalise.

**Comment by PaulioRandall** [comment]
This is a nice start to sidebar elements 👍🏻 

I've renamed the elements in [abstract-sidebar PR](https://github.com/Greater-London-Authority/ldn-viz-tools/pull/106) to match the naming style here.

I notice some of the file names here are _Sidebar_ and others _SideBar_. I've used _Sidebar_ throughout like the original `TabbedSidebar` component.

I'll create a demo branch for next Wednesday's meet that combines both branches. Then we'll have a \[near] whole solution to show and discuss.

**Comment by PaulioRandall** [comment]
[sidebar-components-for-demo](https://github.com/Greater-London-Authority/ldn-viz-tools/tree/sidebar-components-for-demo) combines this PR with [abstract-sidebar](https://github.com/Greater-London-Authority/ldn-viz-tools/pull/106) for demo together.

File: packages/ui/src/lib/sidebar/SidebarSection.svelte [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<script>
+    import SidebarSectionTitle from "./SidebarSectionTitle.svelte";
+    import SidebarDivider from "./SidebarDivider.svelte";
+</script>
+
+<slot />
+
+<SidebarDivider />
```
Comment (PaulioRandall) [inline packages/ui/src/lib/sidebar/SidebarSection.svelte:8]: Do we really need this component? We can just use `<SidebarDivider />` directly to separate content.

**Review by PaulioRandall — APPROVED** [review]

**Pull request closed without merging**